extension/js/common/core/pgp.ts

```diff
@@ -289,6 +289,46 @@ export class Pgp {
     },
   };
 
+  public static friendlyMsgBlockTypeName = (type: MsgBlockType) => {
```

Comment:
Ok, why not. Could be implemented as a dictionary lookup (less code to write), but doesn't matter either way.

Reply:
I also was thinking about dictionary. I don't remember exactly why I choose method implementation of it.

I will replace it with the dictionary in the next commit. It will be better

Reply:
But to be honest, it's not really necessary to have a friendly name for these. It's a bit of a fringe case, so I don't mind showing `signedMsg` as a part of an error, that only happens on one screen.